docs/components/PageHeader.js

```diff
@@ -52,49 +54,50 @@ export default function PageHeader({
         },
       }}
     >
-      <Box marginBottom={2}>
-        <Flex alignItems="baseline" direction="row" gap={2} justifyContent="between" wrap>
-          <Heading>
-            {name}{' '}
-            {badge === 'pilot' ? (
-              <Tooltip
-                inline
-                text={`This is the initial version of ${name}, and additional (non-breaking) functionality is planned for the future. Any feedback is greatly appreciated!`}
+      <Flex direction="column" gap={defaultCode ? 8 : 0}>
+        <Flex direction="column" gap={2}>
+          <Flex alignItems="baseline" direction="row" gap={2} justifyContent="between" wrap>
+            <Heading>
+              {name}{' '}
+              {badge === 'pilot' ? (
```

Comment:
unrelated nit, ignore if you want: this could be a bit simpler/DRYer with a map:
```js
const badgeMap = {
  pilot: { text: ... , tooltipText ... },
  deprecated: { text: ... , tooltipText ... },
};

...

{badge && (
  <Tooltip inline text={badges[badge].tooltipText}>
    <Badge position="top" text={badges[badge].text} />
  </Tooltip>
)}
```
)}

Reply:
DONE